src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String line = ("─".repeat(50));
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        Task[] tasks = new Task[100];
+
+        printHelloMessage();
+
+        Scanner myObj = new Scanner(System.in);
+        int index = 0;
+
+        while (true) {
+            String userInput = myObj.nextLine();
+            String[] userInputArray = userInput.split(" ");
+            String commandWord = userInputArray[0];
+            String commandDescriptor = userInput.substring(commandWord.length()).trim();
+
+            switch (commandWord) {
+            case "bye":
+                printByeMessage();
+                return;
+            case "list":
+                printTasks(tasks, index);
+                break;
+            case "todo":
+                todoHandler(tasks, index, commandDescriptor);
+                index++;
```

Comment:
Maybe it's better to do some modifications to the 'index++' here to get a better abstraction, according to the SLAP.